hail/src/main/scala/is/hail/annotations/RegionValueBuilder.scala

```diff
@@ -505,6 +505,8 @@ class RegionValueBuilder(var region: Region) {
     addRegionValue(t, uis.region, uis.aoff)
   }
 
+  def addAnnotation(t: PType, a: Annotation): Unit = addAnnotation(t.virtualType, a)
```

Comment:
To avoid changing the 70 invocations of addAnnotation

Reply:
This is the wrong signature -- we should either have it be the virtual type, or no type.

Reply:
Done